ui/packages/app/src/hooks/useFeastCoreApi.js

```diff
@@ -19,13 +19,9 @@ export const useFeastCoreApi = (
       baseApiUrl: config.FEAST_CORE_API,
       timeout: config.TIMEOUT,
       useMockData: config.USE_MOCK_DATA,
-      headers: {
-        "Content-Type": "application/json",
-        Authorization: `Bearer ${authCtx.state.idToken}`
-      },
       ...options
     },
-    undefined,
+    authCtx,
```

Comment:
This change has nothing to do with the auth library change and rather corrects something that was (unnecessarily) explicitly setting the auth headers earlier, right?

Reply:
While all the CaraML components were using `access_token` to authenticate users, Feast API used `idToken` JWT. Now, when all the CaraML components use `idToken`, this part can be aligned with `useTuringApi` and `useMerlinApi` hooks.